src/com/ka/spreadsheet/diff/StdoutSpreadSheetDiffCallback.java

```diff
@@ -25,20 +25,16 @@
   private String file1;
   private String file2;
 
-  private CellPos previousCell = null;
-
   @Override
   public void init(String file1, String file2) {
+    super.init(file1, file2);
     this.file1 = file1;
     this.file2 = file2;
```

Comment:
This could also go in the base class

Reply:
Yup, it could.  I was trying to keep the base class clean - data it needed to do the things it wanted to do (_i.e._, contract enforcement).